apps/src/lib/levelbuilder/rubrics/rubricHelper.js

```diff
@@ -42,10 +41,7 @@ export async function saveRubricToTable(
     if (!rubric) {
       navigateToHref(data.redirectUrl);
     } else {
-      setSaveNotificationText(SAVE_COMPLETED_TEXT);
-      setTimeout(() => {
-        setSaveNotificationText('');
-      }, 8500);
+      reload();
```

Comment:
I wonder if you could return the data from server request and update the component without a reload. It would likely mean moving some props into state. We do this in a few of our editors -- the simplest example I could find quickly is [ReferenceGuideEditor](https://github.com/code-dot-org/code-dot-org/blob/ebad91534d0ab2b301c14c5778549c842a04e6e8/apps/src/lib/levelbuilder/reference-guide-editor/ReferenceGuideEditor.jsx#L39)

Reply:
Much nicer and much easier!  Thanks for the tip!